OpenEdXMobile/src/main/java/org/edx/mobile/view/Router.java

```diff
@@ -123,9 +125,14 @@ public void showSplashScreen(Context context) {
         context.startActivity(launchIntent);
     }
 
+    @NonNull
+    public Intent getLogInIntent(@NonNull DeepLink deepLink) {
+        return LoginActivity.newIntent(deepLink);
+    }
+
     @NonNull
     public Intent getLogInIntent() {
-        return LoginActivity.newIntent();
+        return LoginActivity.newIntent(null);
```

Comment:
IMO, the following code is more appropriate. 
```java
    @NonNull
    public Intent getLogInIntent(@Nullable DeepLink deepLink) {
        return LoginActivity.newIntent(deepLink);
    }

    @NonNull
    public Intent getLogInIntent() {
        return getLogInIntent(null);
    }
```

Reply:
done